lib/rubocop/cop/layout/leading_comment_space.rb

```diff
@@ -16,15 +16,25 @@ module Layout
       #
       #   # good
       #   # Some comment
+      #
+      # @example AllowDoxygenCommentStyle: true (default false)
+      #
+      #   # good
+      #   #**
+      #   # Some comment
+      #   # Another line of comment
+      #   #*
+      #
```

Comment:
Could you separate the option example into `AllowDoxygenCommentStyle: false (default)` and `AllowDoxygenCommentStyle: true` as follows?

```ruby
      #
      # @example AllowDoxygenCommentStyle: false (default)
      #
      #   # bad
      #
      #   #**
      #   # Some comment
      #   # Another line of comment
      #   #*
      #
      # @example AllowDoxygenCommentStyle: true
      #
      #   # good
      #
      #   #**
      #   # Some comment
      #   # Another line of comment
      #   #*
      #
```